### PR TITLE
js1-w3 homework finalized

### PR DESCRIPTION
**Pull request by sumanghimire79**

### CODE REVIEW

File: javascript/javascript1/week3/1-Item-Array-Removal/itemArrayRemoval-j1w3.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+nameToRemove = "Ahmad" ;
+for ( let nameIndex in names){
+  if (names[nameIndex] === nameToRemove){
+    console.log( nameToRemove + " found in index of: " + nameIndex);
+    nameToRemove = names.splice(id,1);
+    
+  } else{ console.log( nameToRemove + " not found in index of : "+ nameIndex);
+}
+}
+console.log(names);
```
Comment (esteficodes) [inline javascript/javascript1/week3/1-Item-Array-Removal/itemArrayRemoval-j1w3.js:53]: Ok! 👍 Concerning your question of how to remove more than one element from the array: [here's a useful article](https://love2dev.com/blog/javascript-remove-from-array/).

File: javascript/javascript1/week3/2-when-we-will-be-there/whenWeWillBeThere.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+};
+ 
+function time_convert1(takenTime1) {
+  const hours = Math.floor(takenTime1);
+  const minutes = Math.floor((takenTime1 - hours) * 60);
+  return `${hours} hours and ${minutes} minutes`;
+}
+
+const travelTime1 = travelTimeCalculator(50, 432);
+console.log("Solution two: "+travelTime1); // 8 hours and 38 minutes
```
Comment (esteficodes) [inline javascript/javascript1/week3/2-when-we-will-be-there/whenWeWillBeThere.js:32]: Good! 👍

File: javascript/javascript1/week3/3-series-duration-of-my-life/seriesDurationOfMyLife.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    console.log( " which is " + totalTimeSpent  + " % of my total life.");
+}
+logOutSeriesText(seriesDurations);
+
+/*
+Game of thrones took 0.01% of my life
+Sopranos took 0.012% of my life
+The Wire took 0.007% of my life
+In total that is 0.2% of my life
+*/
```
Comment (esteficodes) [inline javascript/javascript1/week3/3-series-duration-of-my-life/seriesDurationOfMyLife.js:69]: Good!

**Review by esteficodes — APPROVED** [review]
Well done! You may merge your homework. 😸

File: javascript/javascript1/week3/1-Item-Array-Removal/itemArrayRemoval-j1w3.js (hunk 1 above)
Comment (sumanghimire79) [reply javascript/javascript1/week3/1-Item-Array-Removal/itemArrayRemoval-j1w3.js:53]: thank you so much

**Pull request merged**